test/unit/ui/app/actions.spec.js

```diff
@@ -1433,12 +1432,15 @@ describe('Actions', function () {
     let setCurrentLocaleSpy
 
     beforeEach(function () {
-      fetchMock.get('*', enLocale)
+      sinon.stub(global, 'fetch')
+        .returns(Promise.resolve({
+          json: async () => enLocale,
+        }))
     })
 
     afterEach(function () {
       setCurrentLocaleSpy.restore()
-      fetchMock.restore()
+      sinon.restore()
```

Comment:
```suggestion
      global.fetch.restore()
```

Reply:
I think this is more correct, from the docs:

> `var stub = sinon.stub(object, "method");`
>
> Replaces `object.method` with a stub function. An exception is thrown if the property is not already a function.
>
> The original function can be restored by calling `object.method.restore();` (or `stub.restore();`).

Reply:
Well, it's not "more correct", it's doing a different thing. `sinon.restore()` restores everything rather than just this one stub.

It is a bit of a questionable thing to do, I admit. I was considering putting it in the top-level `afterEach` hook instead, but this seemed like a less-dangerous compromise. I could switch it to what you've suggested here for now.